controllers/storagecluster/backingstorageclasses.go

```diff
@@ -0,0 +1,69 @@
+package storagecluster
+
+import (
+	"context"
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	v1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type backingStorageClasses struct{}
+
+// ensureCreated ensures that backing storageclasses are created for the StorageCluster
+func (obj *backingStorageClasses) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.StorageCluster) (reconcile.Result, error) {
+	for i := range sc.Spec.BackingStorageClasses {
+		bsc := &sc.Spec.BackingStorageClasses[i]
+		if bsc.Name == "" {
+			return reconcile.Result{}, fmt.Errorf("backingStorageClass name is empty")
+		}
+		storageClass := &storagev1.StorageClass{}
+		pvReclaimPolicy := corev1.PersistentVolumeReclaimDelete
+		allowVolumeExpansion := true
+		volumeBindingMode := v1.VolumeBindingWaitForFirstConsumer
```

Comment:
These are all constants. just move them above the for loop and take the address for each instance.